src/lib/noop.js

```diff
@@ -0,0 +1,14 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+// Simple helper functions
+// none of these need any external deps
+
+module.exports = function noop() {};
```

Comment:
Actually it's important that `noop` is a singleton so you can use [`=== noop`](https://github.com/plotly/plotly.js/pull/1403/files#diff-621069fcc5090ecff8632dbba197ce0eR77) particularly on the results of `Registry.getComponentMethod` to see if it was found.